src/components/EpisodesNav/EpisodesNav.jsx

```diff
@@ -3,21 +3,58 @@ import {
 	data,
 	EpisodesContext as Episodes
 } from "COMPONENTS/Episodes/";
-import { RENDER } from "./";
+import { s, RENDER } from "./";
 
 export default function EpisodesNav(){
-	
+
+	// CONTEXT
+	// -------------------------------
+	const {
+		get: {
+			episodeCount,
+			activeIndex
+		},
+		set: {
+			activeIndex: setActiveIndex
+		}
+	} = useContext(Episodes);
+
+
+	// RENDER
+	// -------------------------------
+	const isFirstEpisodeActive = activeIndex === 0;
+	const isLastEpisodeActive  = activeIndex === episodeCount - 1;
+
 	const pagination = data.map(RENDER.pagination);
+	
+
+	// CLICK HANDLER
+	// -------------------------------
+	function handleNavPressed(e, newIndex){
```

Comment:
see my earlier comments re: `handleWhatever` function names - this should be `setIndex` or something